pkg/controller/daemon/daemon_controller.go

```diff
@@ -138,17 +141,16 @@ func NewDaemonSetsController(
 	failedPodsBackoff *flowcontrol.Backoff,
 ) (*DaemonSetsController, error) {
 	eventBroadcaster := record.NewBroadcaster()
-	eventBroadcaster.StartStructuredLogging(0)
-	eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: kubeClient.CoreV1().Events("")})
```

Comment:
we were starting to record before running, does this change have some impact , I mean, is there something that can happen between `New` and `Run()` that we should need record the event?

just asking, because this change makes sense to me, but it could be done before, right?

Reply:
I looked through the code and didn't find any place where it would matter.

Yes - it could have been done before...